events-listener/src/main/java/org/hawkular/accounts/events/listener/internal/AccountsMessageListener.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.hawkular.accounts.events.listener.internal;
+
+import org.hawkular.accounts.api.PersonaService;
+import org.hawkular.accounts.api.UserService;
+import org.hawkular.accounts.api.model.Persona;
+import org.hawkular.accounts.events.listener.AccountsEvent;
+
+import javax.annotation.security.PermitAll;
+import javax.ejb.ActivationConfigProperty;
+import javax.ejb.MessageDriven;
+import javax.enterprise.event.Event;
+import javax.inject.Inject;
+import javax.jms.JMSException;
+import javax.jms.Message;
+import javax.jms.MessageListener;
+
+/**
+ * @author Juraci Paixão Kröhling
+ */
+@MessageDriven(activationConfig = {
+        @ActivationConfigProperty(
+                propertyName = "destinationLookup",
+                propertyValue = "topic/HawkularAccountsEvents"
+        ),
+
+        @ActivationConfigProperty(
```

Comment:
The "destination" property is required by ActiveMQ, but not for HornetQ. So, we have a bit of a redundancy here.